src/main/java/me/ryanhamshire/GriefPrevention/Claim.java

```diff
@@ -761,48 +761,15 @@ else if (excludeSubdivisions)
     //used internally to prevent overlaps when creating claims
     boolean overlaps(Claim otherClaim)
     {
-        //NOTE:  if trying to understand this makes your head hurt, don't feel bad - it hurts mine too.
```

Comment:
😂

I remember seeing this when I was working on dev/v20. idk if I just copied this verbatim over or not lol.

Reply:
Oh yeah, I've had a couple chuckles noticing that comment. Didn't click in my head that it could be improved until I did the cube containment check in pistons.